src/karmen_frontend/package.json

```diff
@@ -45,7 +45,7 @@
       "plugin:prettier/recommended"
     ]
   },
-  "version": "0.10.0-rc1",
+  "version": "v0.9.0-rc18",
```

Comment:
And here? Cause we have 0.9.0 rc already deployed in prod so I think we're now in process of preparing 0.10.0, right?